src/index.ts

```diff
@@ -499,6 +499,7 @@ class SpellChecker {
           if (stream.eatWhile(this.rx_word_char)) {
             const word = stream.current().replace(/(^')|('$)/g, '');
             if (
+              word !== '' &&                      // don't check empty words
```

Comment:
How did this bug manifest? Ideally we would first open a bug issue to describe it and then fix it in a separate pull request, adding a changelog entry so that the users can understand what changes were made and why.

Reply:
(not saying that you need to open a bug report now, just what the best practice around JupyterLab repos is; adding a note to the changelog would be good enough).

Reply:
sorry, usually I want to do this, but I thought it is worth to fix it in this release. In the german dictionary if you start with ''  the markdown cells are getting crazy. You see a lot of error messages from Typo.js which is not dealing with empty words. The other bug is that the dictionary was loaded twice when selected, once from the trigger and the second time from the setup change routine ...

Reply:
Can I  update the Changelog after approving?

Reply:
Sure! A good practice would be to request a review from another collaborator after you change anything just to have a another pair of eyes to look at the changes.

Reply:
Which is the best collaborator in this situation? Ask Frederic or Iain?